core/modules/wdb/QueryRunner.h

```diff
@@ -158,7 +158,7 @@ class QueryRunner : public wbase::TaskQueryRunner, public std::enable_shared_fro
 
     /// Used to limit the number of open MySQL connections.
     std::shared_ptr<wcontrol::SqlConnMgr> const _sqlConnMgr;
-    std::atomic<bool> _runQueryCalled{false};
+    std::atomic<bool> _runQueryCalled{false}; ///< If runQuery gets called twice, the scheduler messed up.
```

Comment:
I'm seeing this in the context of the commit called "Added log messages to indicate possible cause for crash". However, I'm not seeing anything in the commit that would do what the commit message says. Was there any intermediate code that got later removed?

Reply:
Could have been.

Reply:
I think the other changes were moved  to a different ticket that has already been merged and these are the differences that remained.